### PR TITLE
[NWPS-1384] GPNRO-Prod Redirect setup & updating prod URL redirection changes

### PR DESCRIPTION
**Pull request by sankar-shunmuga**

### CODE REVIEW

File: repository-data/application/src/main/resources/hcm-content/content/urlrewriter/medical/gpnrotomedicalgpredirects.yaml [hunk 1]
```diff
@@ -68,6 +68,8 @@
       permanent-redirect\" last=\"true\">https://medical.hee.nhs.uk/medical-training-recruitment/medical-specialty-training/general-practice-gp/choose-gp/choose-gp-career-stories/dr-haider-ali</to>\r\
       \n</rule>\r\n\r\n<rule>\r\n  <condition name=\"host\" operator=\"equal\">gprecruitment.hee.nhs.uk</condition>\r\
       \n  <from>^(.*)$</from>\r\n  <to type=\"permanent-redirect\" last=\"true\">https://medical.hee.nhs.uk/medical-training-recruitment/medical-specialty-training/general-practice-gp</to>\r\
+      \n</rule>\r\n\r\n<rule>\r\n  <condition name=\"host\" operator=\"equal\">gprecruitment-prod.hee.nhs.uk</condition>\r\
```
Comment (smanivasag) [inline repository-data/application/src/main/resources/hcm-content/content/urlrewriter/medical/gpnrotomedicalgpredirects.yaml:71]: @sankar-shunmuga - do we need to add other `gprecruitment.hee.nhs.uk` redirects as well for `gprecruitment-prod.hee.nhs.uk` ? thanks

**Review by smanivasag — COMMENTED** [review]
@sankar-shunmuga - Thanks for the PR, it looks good. I have left one comment on `repository-data/application/src/main/resources/hcm-content/content/urlrewriter/medical/gpnrotomedicalgpredirects.yaml`. Could you take a look? thanks

File: repository-data/application/src/main/resources/hcm-content/content/urlrewriter/medical/gpnrotomedicalgpredirects.yaml (hunk 1 above)
Comment (sankar-shunmuga) [reply repository-data/application/src/main/resources/hcm-content/content/urlrewriter/medical/gpnrotomedicalgpredirects.yaml:71]: Email sent to Mark Baldwin. If he comes back, we can create a separate ticket

**Review by smanivasag — APPROVED** [review]
@sankar-shunmuga - Thanks for the PR. Approved! And yes, let's create a separate ticket If Mark B confirm to set up other redirects of `gprecruitment.hee.nhs.uk` for `gprecruitment-prod.hee.nhs.uk`. Thanks!

**Pull request merged**